portfolio/src/main/webapp/script.js

```diff
@@ -80,8 +80,38 @@ function getComments() {
       commentContainer.className += 'comment-container';
       commentContainer.innerText = message;
 
-      document.getElementById('message-container').appendChild(commentContainer);
+      document.getElementById('message-container')
+          .appendChild(commentContainer);
     }
   });
 }
-getComments();
\ No newline at end of file
+getComments();
+
+/**
+ * Checks login status of user, links to login/logout page, hides comment form
+ * if user is logged out.
+ */
+function checkLoginStatus() {
```

Comment:
Function name should describe what it does. `checkLoginStatus` seems like it just reads the login status and potentially returns something but in this case it also changes the DOM.